de.ifw_dresden.nagstamon.yml

```diff
@@ -0,0 +1,145 @@
+app-id: de.ifw_dresden.nagstamon
+runtime: org.freedesktop.Platform
+runtime-version: '18.08'
+sdk: org.freedesktop.Sdk
+command: nagstamon
+finish-args:
+  - --socket=wayland  
+  - --socket=x11
+  - --socket=pulseaudio
+  - --share=network
+  - --filesystem=~/.nagstamon:create
+  - --talk-name=org.freedesktop.Notifications
+modules:
+  - name: cpython
```

Comment:
Is there anything wrong with python3 available in 18.08?

Reply:
I did not know the runtime comes with python3 on it! I took that from a tutorial I found on the internet. I just pushed some changes and I remove this "cphython" module which simplifies and  reduces a lot the time for building the flatpak.